pybamm/parameters/parameter_sets.py

```diff
@@ -156,3 +161,14 @@
     "experiment": "1C_discharge_from_full",
     "citation": "sulzer2019physical",
 }
+
```

Comment:
could you move this up into the lithium-ion section?

Reply:
Sure. I didn't notice there is a classification.